docs/rpc-spec.md

```diff
@@ -662,9 +662,22 @@ from the outside world.
 
 Method name: `port-test`
 
-Request arguments: none
+Request arguments:
+
+| Key | Value Type | Description
+| :-- | :-- | :--
+| `ipProtocol` | number | see below
 
-Response arguments: a Boolean, `port-is-open`
+`ipProtocol` indicates the IP protocol version to be used to check the port status.
+Set to `0` to *only* check for IPv4 port status, set to `1` to *only* check for IPv6 port status,
+or omit it to check if the port is open on *any* of the IP protocol versions.
```

Comment:
0 and 1 aren't very intuitive.

How about a string whose values are `ipv4`, `ipv6`, or `any` where the default is `any`?

Reply:
c618e73